src/game.rs

```diff
@@ -3092,6 +3203,167 @@ mod tests {
         assert!(game.is_err());
     }
 
+    #[test]
+    fn remove_lines() {
```

Comment:
Probably this test is unnecessary because it is covered by the `remove_lines2` test.

Reply:
gone!